src/client/components/ModalWithFeedback/ModalWithFeedback.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import { Button } from '../Button/Button';
+import PropTypes from 'prop-types';
+import './ModalWithFeedback.css';
+import Input from '../Input/Input';
+import { EventCardText } from '../EventCardText/EventCardText';
+import Modal from '../Modal/Modal';
+
+function ModalWithFeedback(props) {
+  const [text, textName] = useState('');
+
+  return (
+    <div className="feedback-modal">
+      <Modal title="Feedback Form">
+        <div className="event-small-container">
+          <span className="dot" />
+          <div className="eventcardtext">
+            <EventCardText
+              title={props.title}
+              datetime={props.datetime}
+              mentor={props.mentor}
+              link={props.link}
+            />
+          </div>
+        </div>
+        <div className="feedback-textfield">
+          <Input
+            type="text"
+            value={text}
+            onChange={(e) => {
+              textName(e.target.value);
+            }}
+            placeholder={props.placeholder}
+          />
+        </div>
+        <div className="feedback-submit-btn">
+          <Button onClick={props.onSubmit}>Submit</Button>
```

Comment:
Please look at the last comment I left: you need to handle this `onClick` _inside_ this component and then inside that handler call `props.onSubmit` with `text` passed as value.
This allows the caller of the modal to do something with that text i.e. save it in DB. With the current code whatever user types into `text` is not available to anyone.